docs/DeveloperDoc.md

```diff
@@ -0,0 +1,15 @@
+# Developer Documentation
+
+## How to render a palette
+
+`Palette.ts` accepts a JSON in a structure demostrated in
```

Comment:
Spelling error "demostrated".

Here is a suggested rewording of the first paragraph up to the third sentence:

"
`Palette.js` constructs a palette based on a JSON file that contains a list of all of the cells in the palette.  An example is found in [`src/keyboards/bmw_palette.json`](../src/keyboards/bmw_palette.json). The `cells` object is the list of all of the cells.  Each cell has a `type` key and an `options` key.
"

The rest of the paragraph is fine.